.env.dev.example

```diff
@@ -42,6 +43,16 @@ EMAIL_HOST_PASSWORD=emailpassword123
 EMAIL_USE_TLS=true
 EMAIL_USE_SSL=false
 
+# S3 configuration
+# Commented are example values if you use Scaleway instead of AWS
+USE_S3=false
+# AWS_ACCESS_KEY_ID=
```

Comment:
Would it make more sense to name these variables `S3_...` since they aren't necessarily AWS?

Reply:
Sadly, these variables are expected by Boto3 (through `django-storages`), so I don't think it can be done. Perhaps we could define `S3_ACCESS_KEY_ID` in `.env` and then "translate" it to `bookwyrm/settings.py` by defining `AWS_ACCESS_KEY_ID = env("S3_ACCESS_KEY_ID")`, but IMO it adds some unwanted complexity.